### PR TITLE
Feature/kzn 10702 prepare for react app vite local development

### PR DESCRIPTION
**Pull request by alexlurvey**
## Goal
"publish" this package so Vite's local dev server added [here](https://github.com/kizen/react-app/pull/5829) only has one file to serve for each library instead of many. There was an issue like this https://github.com/vitejs/vite/issues/3301. React context was simply not working (always returning the default value). This doesn't fix the problem - just gets around it. It's not really worth fixing since you shouldn't need to load TypeScript source files from `node_modules` like we were doing (that was a change by us since it was the fastest way to run edits within this fork).

From now on changes to this repo will need to be built and checked in for the changes to take effect in the consuming apps.
1. Run `yarn build` from the workspaces with changes (`packages/core`, `packages/utils`).
2. This will overwrite `dist/esm/index.js` - commit those changes.

https://kizen.atlassian.net/browse/KZN-10702

### CODE REVIEW

**Review by keegandonley — APPROVED** [review]

**Comment by alexlurvey** [comment]
Don't merge this until after the release. See https://kizen.atlassian.net/browse/KZN-10702?focusedCommentId=78585

**Pull request merged**
